app/src/test/java/at/aau/edu/wizards/MainViewModelTests.kt

```diff
@@ -0,0 +1,91 @@
+package at.aau.edu.wizards
+
+import android.content.SharedPreferences
+import org.junit.Assert.assertEquals
+import org.junit.jupiter.api.Assertions.assertNull
+import org.junit.jupiter.api.Test
+import org.mockito.Mockito
+import org.mockito.kotlin.*
+
+class MainViewModelTests {
+    private val sharedPrefs = Mockito.mock(SharedPreferences::class.java, Mockito.RETURNS_DEEP_STUBS)
```

Comment:
simpler to write, same effect:
`private val sharedPrefs = mock<SharedPreferences>()`